src/plugins/intel_cpu/src/mkldnn_graph_optimizer.cpp

```diff
@@ -16,17 +16,21 @@
 #include "nodes/mkldnn_fake_quantize_node.h"
 #include "nodes/mkldnn_mvn_node.h"
 #include <nodes/mkldnn_transpose_node.h>
+#include <oneapi/dnnl/dnnl_types.h>
 #include "nodes/mkldnn_interpolate_node.h"
 #include "nodes/mkldnn_reduce_node.h"
 #include "nodes/mkldnn_input_node.h"
 #include "nodes/mkldnn_rnn.h"
 #include "nodes/common/cpu_convert.h"
 
 #include "mkldnn/ie_mkldnn.h"
+#include "mkldnn_itt.h"
+#include "memory_desc/cpu_memory_desc_utils.h"
```

Comment:
Didn't get the idea.
Perhaps I should provide more context.
The "ideal" implementation (that I actually tried to implement initially) is to implement the filtering in scope of canFuse() method. This would automatically support new transformations as you mentioned. Maybe this is the same idea you was talking about.
But the problem is that this method is not used in several transformations and aligning transformations behavior is seemed like quite a big refactoring.
So the decision was to implement it via those code inserts.
Let me know you if meant something different

Reply:
Ok, since we can not afford the transformations refactoring, let's leave the solution as it is.